pages/vi/nextcloud-tor.md

```diff
@@ -12,8 +12,10 @@
 
 ## Prepare Treehouses image
 
-Download the latest treehouses image from http://download.treehouses.io, then use [balenaEtcher](https://etcher.io) to flash the image onto your SD card.  
-
+Download the latest treehouses image from http://download.treehouses.io, then use [balenaEtcher](https://etcher.io) to flash the image onto your SD card.
+```
+Use a new the SD cards (Don't use the one that used for step 1)
+```
```

Comment:
This should be not be a code block as its not something a user would input into their system.
This would best fit as a warning:

`Warning: Use a new SD Card for these steps; otherwise you will encounter errors.`

`Warning: `, `Note:`, and `Hint:` all render differently which which help to highlight and provide the relevant and associated information.